pretyping/nativenorm.ml

```diff
@@ -132,15 +132,15 @@ let construct_of_constr_notnative const env tag (mind, _ as ind) u allargs =
   (mkApp(mkConstructU((ind,i),u), params), ctyp)
  
 
-let construct_of_constr const env tag typ =
+let construct_of_constr const env sigma tag typ =
```

Comment:
@SkySkimmer did you check if `vnorm` needs a similar fix?

Reply:
vnorm does
~~~ocaml
  (** This evar-normalizes terms beforehand *)
  let c = EConstr.to_constr ~abort_on_undefined_evars:false sigma c in
  let t = EConstr.to_constr ~abort_on_undefined_evars:false sigma t in
~~~
so no.
I'm going to add vm_compute to that one test file for future safety.